### ISSUE TITLE
Resource Landing Page: Rendering of readme.txt files is ignoring formatting

### ISSUE THREAD

**Issue by horsburgh** [hydroshare/hydroshare#3176]
I suspect this is an issue with line ending characters. For example, see the following resource:

https://www.hydroshare.org/resource/a3a9ba772aac4cba9533b35bb6b5fe42/

HydroShare's rendering:
![image](https://user-images.githubusercontent.com/3485902/53189064-e3e76b80-35c3-11e9-80a1-feccfcb154a9.png)

What it should look like:
![image](https://user-images.githubusercontent.com/3485902/53189138-0f6a5600-35c4-11e9-8be3-1af07ffb6387.png)

**Comment by Maurier** [comment]
The problem is in the container we use to display this content. We are using an HTML `div` tag which ignores the format. A `pre` tag correctly renders the text with its format. I will test and make sure this does not conflict when rendering a readme.md file and submit a PR soon.

**Comment by horsburgh** [comment]
Yeah - rendering of readme.md looks good to me in all of the file's I've tested so far. It's just the .txt files.
